$NetBSD$

  - Added missing entry to avoid 'Error: 'GLint' not in typemap in OpenGL.xs'

--- typemap.orig	2005-10-05 11:25:49.000000000 +0200
+++ typemap
@@ -54,6 +54,7 @@ GLboolean		T_U_CHAR
 GLUnurbsObj *		T_PTR
 GLUtesselator *		T_PTR
 GLuint			T_UV
+GLint			T_IV
 GLint*			T_PTR
 float *			T_PTR
 float			T_NV
